src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,138 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
-
+import PropTypes from 'prop-types';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { getFilm } from '../../api/api';
+
+export const FindMovie = ({ movies, addMovie }) => {
+  const [query, setQuery] = useState('');
+  const [newFilm, setNewFilm] = useState({});
+  const [error, setError] = useState(false);
+  const [preview, setPreview] = useState(false);
+  const [addingError, setAddingError] = useState(false);
+
+  const handleChange = (event) => {
+    setQuery(event.target.value);
+    setError(false);
+  };
+
+  const searchFilm = async() => {
+    const request = await getFilm(query);
+
+    if (request.Response === 'True') {
+      setNewFilm({
+        title: request.Title,
+        description: request.Plot,
+        imgUrl: request.Poster,
+        imdbUrl: `https://www.imdb.com/title/${request.imdbID}/`,
+        imdbId: request.imdbID,
+      });
+      setError(false);
+      setPreview(true);
+      setAddingError(false);
+    } else {
+      setError(true);
+      setPreview(false);
```

Comment:
if we found a movie and after tried to find another, we need to delete the previous from the state because we can add the previous movie when the current response is false

Reply:
fixed